From 27dccc04aa5148e8be7ebb4edef0ee1fd6ac1aea Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Wed, 22 Dec 2021 06:19:08 +0200
Subject: [PATCH] [SW-0] gaudi2: preboot: increase sram limit

This patch increases the SRAM limit of Preboot image to accommodate
the increase in lodable image size due to upcoming changes in preboot
related to root of trust.

Change-Id: Id0084c7a19d2b28110299844ba63603c60842c65
---

diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index ceebcda..9ca6b24 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -9,7 +9,7 @@
 #include <bitops.h>
 
 #define PPBOOT_SIZE			0x4800		/* 16KB */
-#define PREBOOT_SIZE			0xFC00		/* 63KB */
+#define PREBOOT_SIZE			0x1FC00		/* 127KB */
 
 #define HL_FW_BOOT_PTRS_BASE		0x80000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
